### PR TITLE
added info about version of the program

### PR DESCRIPTION
**Pull request by StrikerRUS**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
     description="Code-generation for various ML models into native code.",
     long_description=long_description,
     long_description_content_type="text/markdown",
     license="MIT",
-    packages=find_packages(exclude=["tests.*", "tests"]),
-    package_data={
-        "": ["linear_algebra.java", "linear_algebra.c", "linear_algebra.go",
-             "linear_algebra.js"],
-    },
+    packages=find_packages(exclude=["tests.*", "tests", "tools"]),
```
Comment (StrikerRUS) [inline setup.py]: I think that `tools` should be included into `m2cgen`'s installation process. Please correct me if I'm wrong.

Comment (izeigerman) [reply setup.py]: No, I don't think so, why? `tools` are dev tools and I don't see why they should be a part of the installation process.

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,4 @@
+include LICENSE
+recursive-include m2cgen VERSION.txt
+recursive-include m2cgen linear_algebra.*
+global-exclude *.py[cod]
```
Comment (izeigerman) [inline MANIFEST.in]: Is it smart enough to exclude all the pycache directories as well?

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1,4 @@
+include LICENSE
+recursive-include m2cgen VERSION.txt
```
Comment (izeigerman) [inline MANIFEST.in]: Why recursive?

**Comment by izeigerman** [comment]
Thank you for this. A few questions + don't think `tools` should be included and let's merge it.

File: setup.py (hunk 1 above)
Comment (StrikerRUS) [reply setup.py]: Oh, I'm so sorry, I forgot to type **not**! I excluded them in code but failed to comment that 😢 .

File: MANIFEST.in (hunk 2 above)
Comment (StrikerRUS) [reply MANIFEST.in]: It was taken from the official documentation. I think we can trust them.
https://packaging.python.org/guides/using-manifest-in/ (the very bottom of the page)

File: MANIFEST.in (hunk 3 above)
Comment (StrikerRUS) [reply MANIFEST.in]: I think it is the syntax to include filed from directories. 

> 

recursive-include dir-pattern pat1 pat2 ... | Include all files under directories matching dir-pattern that match any of the listed patterns
-- | --




https://packaging.python.org/guides/using-manifest-in/#manifest-in-commands

I can be wrong, but `include` works only for root-located files...

File: setup.py (hunk 1 above)
Comment (izeigerman) [reply setup.py]: Ah, makes sense now. It only now occurred to me that the comment contradicted the code :)

**Pull request merged**